dyn_em/solve_em.F

```diff
@@ -3799,6 +3799,7 @@ SUBROUTINE solve_em ( grid , config_flags  &
       &        ,height=grid%height                                         &
       &        ,tempc=grid%tempc                                         &
       &        ,ccn_conc=grid%ccn_conc                                   & ! RAS
+      &        ,sbmradar=sbmradar,num_sbmradar=num_sbmradar              & ! for SBM
```

Comment:
@JS-WRF-SBM 
This is the ONLY line that should be changed in solve_em.F. There are too many white space changes. Those are going to cause conflicts, and be a real pain to resolve.